packages/accounts-controller/src/AccountsController.test.ts

```diff
@@ -1727,7 +1728,8 @@ describe('AccountsController', () => {
       ).toStrictEqual(mockAccount2.id);
     });
 
-    it("should set the selected account to '' if the account is not found", () => {
+    it('should throw error if the account is not found', () => {
```

Comment:
After re-reading this, we can remove this test no? It's the same than "should throw an error for an unknown account ID"?